client/components/submission.jsx

```diff
@@ -64,6 +64,10 @@ export default function Submission(props) {
     setMessageBox(false);
   };
 
+  const openURL = () => {
+    window.open(props.url);
+  };
```

Comment:
Is there a way for you to use a legit link element instead of this? Does the `IconButton` let you do that? It's not good to use buttons to open links.